core/os/main_loop.cpp

```diff
@@ -61,6 +61,8 @@ void MainLoop::_bind_methods() {
 	BIND_CONSTANT(NOTIFICATION_WM_ABOUT);
 	BIND_CONSTANT(NOTIFICATION_CRASH);
 	BIND_CONSTANT(NOTIFICATION_OS_IME_UPDATE);
+
+	ADD_SIGNAL(MethodInfo("_permission_results", PropertyInfo(Variant::STRING, "permission"), PropertyInfo(Variant::BOOL, "granted")));
```

Comment:
Signals shouldn't be prefixed with `_`, unless you don't want them to be visible for users. But since this signal is not used for anything in this PR, I assume you add it so that users can connect to it?

Reply:
Yes, users can connect to it. I fixed it.